android/modules/xml/src/java/ti/modules/titanium/xml/XMLModule.java

```diff
@@ -25,6 +25,7 @@
 import org.appcelerator.kroll.common.Log;
 import org.xml.sax.SAXException;
 
+// DEPRECATED IN SDK 8.0.0
```

Comment:
Might as well make use of the actual @deprecated annotation available in Java: https://stackoverflow.com/questions/9031832/how-to-declare-or-mark-a-java-method-as-deprecated

```java
/**
 * @deprecated Will be removed from core SDK in 9.0.0 and moved out to a native module
 **/
@Deprecated
@Kroll.module
public class XMLModule extends KrollModule
```